bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/Navigation.java

```diff
@@ -72,6 +72,12 @@
      */
     String PN_STRUCTURE_DEPTH = "structureDepth";
 
+    /**
+     * Name of the resource property that for redirecting pages will indicate if original page or redirect target page should be returned.
+     * Property is false by default.
+     * If `true` - original page is returned. If `false` or not configured - redirect target page.
+     */
+    String PN_DISABLE_SHADOWING = "disableShadowing";
```

Comment:
I would rather we only define the property once and reuse it in other components.

Reply:
I am not sure where can I move it. There is no dedicated constants class yet, should I create something like ComponentConstants or ComponentConfigConstants?